llvm/test/CodeGen/NVPTX/load-store.ll

```diff
@@ -990,5 +1684,10 @@ define void @local_unordered_volatile(ptr addrspace(5) %a, ptr addrspace(5) %b,
   ; CHECK: st.local.f64 [%rd{{[0-9]+}}], %fd{{[0-9]+}}
   store atomic volatile double %f.add, ptr addrspace(5) %e unordered, align 8
 
+  ; TODO: LLVM IR Verifier does not support atomics on vector types.
+
   ret void
 }
+
+; TODO: missing .const statespace tests
```

Comment:
Nit: `add missing <X/Y/Z> tests...`

Reply:
Have redone the comments throughout to be clearer.